### ISSUE TITLE
question about train_data and validation_data 

### ISSUE THREAD

**Issue by LiuTingWed** [quark0/darts#145]
hi
I have some problem with train_data and validation_data.
I want to run this algorithm(Darts) to datasets A ,which is have 2000 picture, validate to dataset B, which is have 400 picture,
However, I noticed that in your code about the setting of the validation set, you split the training set in two and use a part for validation. Sorry, this part of the gradient calculation is not too clear to me.
so, how can i run this algorithm(Darts) to datasets A , validate to dataset B ,
any suggestion will appreciate
